raft.go

```diff
@@ -190,6 +190,12 @@ func (r *Raft) runFollower() {
 		case b := <-r.bootstrapCh:
 			b.respond(r.liveBootstrap(b.configuration))
 
+		case <-r.leaderNotifyCh:
```

Comment:
Why do we need to explicitly ignore a notification from this channel now?

Reply:
I'm just being defensive, since the channels are buffered, and thinking about the case where we've transitioned from leader to follower and there might be a notification in the channel.  Probably not needed, but also doesn't hurt.

Reply:
Definitely, makes sense!